GoogleForCreators/web-stories-wp
# Fix selecting media creating history entries

## Summary



## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions
1. Add Media element and save to disable "Save draft" button.
2. Now click on media.
3. Verify that the "Save draft" button did not change to active.
4. Try a few times just in case.


---



Fixes #2664
